2024-06-26 13:28:00.481 [info] 




2024-06-26 13:28:00.503 [info] Extension host agent started.
2024-06-26 13:28:00.807 [info] [<unknown>][ee90d82e][ManagementConnection] New connection established.
2024-06-26 13:28:00.809 [info] [<unknown>][d086467f][ExtensionHostConnection] New connection established.
2024-06-26 13:28:00.944 [info] [<unknown>][d086467f][ExtensionHostConnection] <2266> Launched Extension Host Process.
2024-06-26 13:28:54.182 [error] [File Watcher (parcel)] Unexpected error: inotify_add_watch on '/home/azureuser/.vscode-server/data/logs/20240623T100010/exthost1/ms-python.debugpy' failed: No such file or directory (EUNKNOWN) (path: /home/azureuser)
2024-06-26 13:28:54.183 [error] [File Watcher (universal)] restarting watcher from error in watch request (retrying request): inotify_add_watch on '/home/azureuser/.vscode-server/data/logs/20240623T100010/exthost1/ms-python.debugpy' failed: No such file or directory ({"path":"/home/azureuser","excludes":["**/.git/objects/**","**/.git/subtree-cache/**","**/node_modules/*/**","**/.hg/store/**","/home/azureuser/.vscode-server/extensions/**"],"recursive":true})
2024-06-26 13:33:00.504 [info] New EH opened, aborting shutdown
